Name the example inf iterator of iterators sequence (#22)
tc39/proposal-iterator-sequencing: README.md

let digits = Iterator.concat(lows, [4, 5], highs);
```

> For the (rare) case of infinite iterators of iterators, use the existing [`flatMap`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/flatMap) with the identity function.

Reviewer: ```suggestion
For the (rare) case of infinite iterators of iterators, use [`Iterator.prototype.flatMap`](https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-iterator.prototype.flatmap) with the identity function.
```

I'd rather link to the spec for flatMap if it must have a link.

```js

> function* self_counting_sequence_helper() {

Author: ```suggestion
function* selfCountingSequenceHelper() {
```

  for (let n = 1;; ++n) {
    yield Array(n).fill(n);
  }
}

> let self_counting_sequence = self_counting_sequence_helper().flatMap(x => x)

Author: ```suggestion
let selfCountingSequence = selfCountingSequenceHelper().flatMap(x => x)
```
